Formula/mingw-w64.rb

```diff
@@ -79,7 +79,7 @@ def install
         --target=#{target}
         --with-sysroot=#{arch_dir}
         --prefix=#{arch_dir}
-        --with-bugurl=https://github.com/Homebrew/homebrew-core/issues
+        --with-bugurl=#{CoreTap.instance.issues_url}
```

Comment:
Just thinking about the `brew extract` case. Could this be like this? Not sure if it works.
```suggestion
        --with-bugurl=#{tap.issues_url}
```

Reply:
Yep, that works, nice idea, thank you!